developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -1649,6 +1649,40 @@ end
 ----
 
 
+[[extending-hostgroup-ui]]
+=== Extending hostgroup UI
+
+[[extending-hostgroup-ui-index-actions]]
+==== Adding actions to the index page
+A plugin can add items to the `Actions` dropdown in the table on the hostgroups overview page.
+
+Adding an item to the actions dropdown requires adding a helper with a method that accepts a hostgroup as an argument and
+returns a list of hashes, where each hash will
+have two predefined fields: `:action` and `:priority`. The `:priority` value would be used by the
+system to define the order of the items to show. The lower the priority, the
+higher the item will show.
+
+In plugin helper (`my_plugin_helper.rb`):
+[source, ruby]
+----
+def my_plugin_hostgroups_actions(hostgroup)
+  [
+    { :action => [_('new_action'), other_properties], :priority => 20 }
```

Comment:
```suggestion
    { :action => display_link_if_authorized('foo', {bar: 'bar'}, {baz: 'baz'}), :priority => 20 }
```
The item should be an HTML element (probably link) that will be embedded as an item in the dropdown

Reply:
You are right, I need to update the documentation. Thanks!